### PR TITLE
Documentations

### PR DESCRIPTION
**Pull request by Shea690901**
I've updated the sample configuration…
Within 'local_options.README' I've two macros marked as to check (very end of the file).
For one of them (RUNTIME_LOADING) I've already filed an issue (#287) because I couldn't find the relevant function…

### CODE REVIEW

**Comment by Shea690901** [comment]
And just after initiating this request I've seen your answer to #287…
Here the updated version without RUNTIME_LOADING (and one macro I missed as outdated)

**Comment by Shea690901** [comment]
And a minor bug in the build process found:
the check for outdated macros within local_options was missing a '-w' for grep…

File: src/vm/internal/compiler/lex.cc [hunk 1]
```diff
@@ -2794,9 +2794,7 @@ void add_predefines() {
   add_quoted_predefine("__OPTIMIZATION__", OPTIMIZE);
 
 /* Backwards Compat */
-#ifndef CDLIB
```
Comment (thefallentree) [inline src/vm/internal/compiler/lex.cc]: why did you remove this ?

File: src/Config.example [hunk 2]
```diff
@@ -91,68 +106,227 @@ maximum array size : 15000
...
-#          The following aren't currently used or implemented (yet)           #
-###############################################################################
+# There is a hash table for living objects, used by find_living().
+# This must be one of 4, 16, 64, 256, 1024, 4096
+living hash table size : 256
+
+#
+gametick msec : 1000
+
+# Set heartbeat interval in microseconds (us).
```
Comment (thefallentree) [inline src/Config.example]: the docs here is obviously wrong. This config entry has changed unit type to "msec" . please delete all these obsolete lines..

**Comment by thefallentree** [comment]
Thanks for your PR, can you please squash them and address the above issues?

File: src/vm/internal/compiler/lex.cc (hunk 1 above)
Comment (Shea690901) [inline src/vm/internal/compiler/lex.cc]: Because I couldn't find any other references to thís macro…
And I don't think anyone would use an unchanged CD-lib with either MudOS or FluffOS, something which wouldn't work anyway.
The first 'glitch' would be a master which couldn't be loaded because MudOS is missing some essential applies (master::inaugurate_master at the front).

**Comment by Shea690901** [comment]
I've updated the comments for heartbeat from μs to ms…
Concening the squashing: this the way you wanted it?

[Note to self: be more careful when rebasing!, otherwise you get the same commit at least two or three times into your history!]

File: src/Config.example [hunk 3]
```diff
@@ -91,68 +106,227 @@ maximum array size : 15000
...
+
+#
+gametick msec : 1000
+
+# Set heartbeat interval in milliseconds (ms).
+# 1,000 ms = 1 second.  The value of this option specifies the frequency with
+# which the heart_beat method will be called in those LPC objects which have
+# called set_heart_beat(1).
+#
+# [NOTE: if ualarm() isn't available, alarm() is used instead.  Since alarm()
```
Comment (thefallentree) [inline src/Config.example]: this whole paragraph is obsolete too, as we don't use ualarm anymore.

File: src/outdated_define.txt [hunk 4]
```diff
@@ -27,3 +27,26 @@
  OLD_RANGE_BEHAVIOR
  WARN_OLD_RANGE_BEHAVIOR
  SUPPRESS_ARGUMENT_WARNINGS
+
```
Comment (thefallentree) [inline src/outdated_define.txt]: no need for space here

**Comment by thefallentree** [comment]
LGTM otherwise, thanks!

File: src/Config.example (hunk 3 above)
Comment (Shea690901) [inline src/Config.example]: Is removed now… (or not added at all, depending on view point)

**Comment by Shea690901** [comment]
Both issues updated…
Also added a förgotten line concerníng  *gametick  msec*

File: doc/applies/master/get_mud_stats.4 [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+.SH DESCRIPTION
+This master apply is called by the driver every time a client requests MSSP
+infos when doing telopt.
+
+.SH SEE ALSO
+.UR http://tintin.sourceforge.net/mssp/
+The MSSP Protocol
+.UE ,
+.UR http://lpmuds.net/smf/index.php?topic=943.0
+MSSP on lpmids.net
```
Comment (thefallentree) [inline doc/applies/master/get_mud_stats.4]: is this a typo ?

File: src/Config.example [hunk 6]
```diff
@@ -7,24 +7,18 @@
 ###############################################################################
 
 # name of this mud
+# should be changed prioir to the 'mud ip', otherwise it might yield strange
+# results to either yourself or other users of this mudlib!
 name : testsuite
 
 # for machines with multiple IP addresses, this specifies which one to use.  this
 # replaces the SERVER_IP compile-time define.
 #mud ip : 255.255.255.255
```
Comment (thefallentree) [inline src/Config.example]: we should remove this line...

File: doc/applies/master/get_mud_stats.4 (hunk 5 above)
Comment (Shea690901) [inline doc/applies/master/get_mud_stats.4]: Ups…

**Comment by Shea690901** [comment]
While looking more closely into the source (I'm currently writing man pages for efun::db_\*) I found that the entry for *USE_MYSQL* and the like no longer is valid…
On this note I added *DEFAULT_DB* without further comment even though from viewpoint source it's optional (lines 260-264 in [db.cc](https://github.com/fluffos/fluffos/blob/next-3.0/src/packages/db/db.cc))
